README.md

```diff
@@ -188,6 +188,10 @@ NOTES:
 
 ## Release notes
 
+### 3.17
+
++ Add support for the new filter's format
```

Comment:
Please, also add information about supporting multiple filters. Maybe we should add an example to the readme as well